spec/ameba/rule/documentation/documentation_spec.cr

```diff
@@ -1,14 +1,14 @@
 require "../../../spec_helper"
 
 module Ameba::Rule::Documentation
-  subject = Documentation.new
-    .tap(&.ignore_classes = false)
-    .tap(&.ignore_modules = false)
-    .tap(&.ignore_enums = false)
-    .tap(&.ignore_defs = false)
-    .tap(&.ignore_macros = false)
-
   describe Documentation do
+    subject = Documentation.new
```

Comment:
only now noticed that the rule name is kinda off:

`Ameba::Rule::Documentation::Documentation`